### DISCUSSION TITLE
Did v0.10 get rid of noerrorbells and novisualbell?

### DISCUSSION THREAD

**Discussion by fbunt** [neovim/neovim#28899] [Q&A] [answered]
I have had the following in my init.lua file for a while now:

```lua
vim.o.noerrorbells = true
vim.o.novisualbell = true
```

After updating to 0.10, I now get the following error when opening nvim:

```
Error detected while processing /home/fred/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/fred/.config/nvim/init.lua:1: Unknown option 'noerrorbells'
stack traceback:
        [C]: in function '__newindex'
        /home/fred/.config/nvim/init.lua:1: in main chunk
```

`noerrorbells` is still in the help pages so I assumed it was still valid. Am I running into a bug or did I miss something in the release notes?

**Comment by bfredl** [comment, marked as answer]
hmm, it's a bit weird that this ever worked but given they are boolean options probably what you want is
```lua
vim.o.errorbells = false
vim.o.visualbell = false
```

**Comment by fbunt** [comment]
I just tried it with some other `no{option_flag}` settings and it seems like v0.10 has stopped supporting all of them. I'll update my config.

**Reply by wookayin** [reply]
Which versions were you using? In none of neovim 0.6, 0.7, 0.8, and 0.9, `vim.o.no{option} = ...` raises the same error "unknown option" for me.

**Reply by fbunt** [reply]
I was using 0.9.

**Comment by fbunt** [comment]
Should there maybe be a note in the release notes about this?

**Reply by clason** [reply]
There is. (To the extent that this is an actual change.)

**Reply by fbunt** [reply]
OK thanks
